modules/build/src/main/scala/scala/build/input/Inputs.scala

```diff
@@ -466,7 +466,7 @@ object Inputs {
       elements = Nil,
       defaultMainClassElement = None,
       workspace = workspace,
-      baseProjectName = workspace.baseName,
+      baseProjectName = baseName(workspace),
```

Comment:
no need to extract this into a function, it's not reused anywhere.
```suggestion
      baseProjectName = if workspace == os.root then "" else workspace.baseName,
```

Reply:
It's also used on line 156.